android/app/src/main/java/org/fossasia/openevent/core/feed/FeedFragment.java

```diff
@@ -0,0 +1,64 @@
+package org.fossasia.openevent.core.feed;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.design.widget.TabLayout;
+import android.support.v4.view.ViewPager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.common.ui.base.BaseFragment;
+import org.fossasia.openevent.common.ui.image.OnImageZoomListener;
+import org.fossasia.openevent.core.feed.facebook.FacebookFeedAdapter;
+import org.fossasia.openevent.core.feed.facebook.FacebookFeedFragment;
+import org.fossasia.openevent.core.feed.twitter.TwitterFeedFragment;
+
+import butterknife.BindView;
+
+public class FeedFragment extends BaseFragment {
+
+    @BindView(R.id.viewpager)
+    protected ViewPager viewPager;
+    @BindView(R.id.tabLayout)
+    protected TabLayout feedTabLayout;
+
+    private FacebookFeedAdapter.OpenCommentsDialogListener openCommentsDialogListener;
```

Comment:
If it is general, why is it inside FacebookFeedAdapter

Reply:
@iamareebjamal 
Presently only Facebook comments are shown. Twitter feed doesn't support comments as of now.
Should I move it to a general location and make an issue for adding comments to twitter feed?

Reply:
If it is used in FeedFragment, it should be in general package, let's not show twitter comments but just move it to general package